services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling the exception when a movie session with the given ID does not exist before attempting to delete it. Use a try-except block to catch `MovieSession.DoesNotExist`.